src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-
+    
+    transition: transform 0.3s, color $trans-duration;
+    
+    
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
Create variables for repeating colors

Reply:
i have no repeating colors

Reply:
But you should create variables for colors
Bc in the production you will always do this & use it)